frontend/alert/components/managealert/ToggleManager.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+import { Flex, FlexProps } from "pcx-shared-components/src/common/layout";
+import { Img, P } from "../common/common";
+import { AlertAction, WrappedStyled } from "../../types";
+import ToggleSwitch from "../common/ToggleSwitch";
+
+type ActionFlexProps = FlexProps & {
+    background: string;
+};
+
+// Component associated with Resubscribe and Cancel buttons
+// for each alert
+interface ToggleManagerProps {
```

Comment:
nit: I think this comment is outdated since we removed resubscribe and cancel buttons.